pytorch/training/docker/1.6.0/py3/Dockerfile.cpu

```diff
@@ -0,0 +1,125 @@
+ENV DGLBACKEND=pytorch
+
+ARG PT_TRAINING_URL=https://aws-pytorch-binaries.s3-us-west-2.amazonaws.com/r1.6.0_aws/20200723-225547/9d43d8c6ffb8b6661f2dd1b9b393d7e77a28c59d/cpu/torch-1.6.0-cp36-cp36m-manylinux1_x86_64.whl
+WORKDIR /
+
+RUN apt-get update \
+ && apt-get install -y --no-install-recommends \
+    ca-certificates \
+    cmake \
+    curl \
+    emacs \
+    git \
+    wget \
+    vim \
+    build-essential \
+    jq \
+    libglib2.0-0 \
+    libgl1-mesa-glx \
+    libsm6 \
+    libxext6 \
+    libxrender-dev \
+    zlib1g-dev \
+    software-properties-common \
+ && rm -rf /var/lib/apt/lists/*
+
+RUN wget https://www.open-mpi.org/software/ompi/v4.0/downloads/openmpi-$OPEN_MPI_VERSION.tar.gz \
+ && gunzip -c openmpi-$OPEN_MPI_VERSION.tar.gz | tar xf - \
+ && cd openmpi-$OPEN_MPI_VERSION \
+ && ./configure --prefix=/home/.openmpi \
+ && make all install \
+ && cd .. \
+ && rm openmpi-$OPEN_MPI_VERSION.tar.gz \
+ && rm -rf openmpi-$OPEN_MPI_VERSION
+
+# The ENV variables declared below are changed in the previous section
+# Grouping these ENV variables in the first section causes
+# ompi_info to fail. This is only observed in CPU containers
+ENV PATH="$PATH:/home/.openmpi/bin"
+ENV LD_LIBRARY_PATH="$LD_LIBRARY_PATH:/home/.openmpi/lib/"
+RUN ompi_info --parsable --all | grep mpi_built_with_cuda_support:value
+
+# Copy workaround script for incorrect hostname
+COPY changehostname.c /
+COPY start_with_right_hostname.sh /usr/local/bin/start_with_right_hostname.sh
+
+RUN curl -L -o ~/miniconda.sh https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh \
+ && chmod +x ~/miniconda.sh \
+ && ~/miniconda.sh -b -p /opt/conda \
+ && rm ~/miniconda.sh \
+ && /opt/conda/bin/conda install -y -c anaconda \
+    python=$PYTHON_VERSION \
+    numpy==1.16.4 \
+    ipython==7.10.1 \
+    mkl==2019.4 \
+    mkl-include==2019.4 \
+    cython==0.29.12 \
+    typing==3.6.4 \
+    "pyopenssl>=17.5.0" \
+ && conda install -c conda-forge \
+    opencv==4.0.1 \
+ && conda install -y \
+    scikit-learn==0.21.2 \
+    pandas==0.25.0 \
+    h5py==2.9.0 \
+    requests==2.22.0 \
+ && conda install -c dglteam -y dgl==0.4.3 \
+ && /opt/conda/bin/conda clean -ya \
+ && conda clean -ya
+
+# The following section uninstalls torch and torchvision before installing the
+# custom versions from an S3 bucket. This will need to be removed in the future
+RUN pip install --upgrade pip --trusted-host pypi.org --trusted-host files.pythonhosted.org \
+ && ln -s /opt/conda/bin/pip /usr/local/bin/pip3 \
+ && pip install --no-cache-dir -U \
+    "awscli<2" \
+    fastai==1.0.59 \
+    scipy==1.2.2 \
+    smdebug==${SMDEBUG_VERSION} \
+    "sagemaker<2" \
+    sagemaker-experiments==0.1.7 \
+    "sagemaker-pytorch-training<3" \
+ && pip install --no-cache-dir -U ${PT_TRAINING_URL} \
+ && pip uninstall -y torchvision \
+ && pip install --no-deps --no-cache-dir -U \
+    https://torchvision-build.s3.amazonaws.com/1.5.1/cpu/torchvision-0.6.0-cp36-cp36m-linux_x86_64.whl
```

Comment:
is this supposed to be 1.5.1?

Reply:
Same comment as above. Will address once we have a final binary.

Reply:
ok thanks for the info, was just confused why one was 1.5.0 and the other was 1.5.1